person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```

Comment:
- [ ] I think it would be great if you move this method to be before the of_age? method since you add the keyword private so this will make all the method after it private but we don't need this to be private just the method of_age? it should be private 🚀💪🏻🔥